cl/api/models.py

```diff
@@ -1,10 +1,64 @@
+from enum import Enum
+
 from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from rest_framework.authtoken.models import Token
 
+from cl.lib.models import AbstractDateTimeModel
+
 
 @receiver(post_save, sender=settings.AUTH_USER_MODEL)
 def create_auth_token(sender, instance=None, created=False, **kwargs):
     if created:
         Token.objects.create(user=instance)
+
+
+class WebhookEventType(Enum):
+    RECAP_EMAIL = "Recap Email"
+
+    @classmethod
+    def choices(cls):
+        return [
+            (
+                key.value,
+                key.value,
+            )
+            for key in cls
+        ]
```

Comment:
Take a look at the docs here for how to do enums in Django: https://docs.djangoproject.com/en/3.1/ref/models/fields/#field-choices-enum-types

Reply:
I'd also use the IntegerField for this, so something like:

```
class WebhookEventType(models.IntegerChoices):
    RECAP_EMAIL = 1, "RECAP email received"
    ALERT = 2, "Alert triggered"
```

I think that's all it needs.